CONTRIBUTING.md

```
@@ -374,6 +374,28 @@ Every negative test follows the same four-step pattern:
 | `Then it should fail` | Step failed (any reason, no message check). |
 
+## Step definition conventions
+
+`scripts/docs.php` validates step definitions against the conventions enforced
+in 6.0. CI fails on any violation. Before pushing, run `ahoy docs` to
+regenerate `STEPS.md` and surface any issues.
+
+| Rule | Applies to |
+|------|------------|
+| `@Given` steps ending with `:` must contain `following`. | Table-form Givens |
+| `@When` steps must contain `I `. | All Whens |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)